azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -47,13 +47,14 @@ stages:
               templateLocation: 'Linked artifact'
               deploymentMode: 'Incremental'
               deploymentOutputs: 'OUTPUTS'
-          - task: 'AzureFunctionApp@1'
-            inputs:
-              appName: '$(environment)-fn-$(functionAppName)-$(location)'
-              appType: 'functionAppLinux'
-              azureSubscription: '$(serviceConnection)'
-              package: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
-              deployMethod: 'zipDeploy'
+          - script:
+              pipeline_ip=$(curl -s ifconfig.me/ip)
+              pipeline_firewall_name=PipelineFirewall
```

Comment:
I'd suggest you add a timestamp here, concatenated with the name, or any pipeline hash or unique ID. Because you can have more than 1 pipeline running at the same time, and we don't want to remove the rule from one pipeline in another.

Reply:
Okay, I'll concatenate with a timestamp.